Modelica/Mechanics/Rotational.mo

```diff
@@ -6709,7 +6709,7 @@ Negative torque brakes in positive direction of rotation, but accelerates in rev
       extends Rotational.Interfaces.PartialTorque;
       import Modelica.Constants.pi;
       parameter Modelica.SIunits.Torque tau_constant
-        "Constant torque (if negative, torque is acting as load in positive direction of rotation)";
+        "Constant torque (if negative, torque is acting as load)";
```

Comment:
Should it be `tau_nominal` and "Nominal torque" to match the SignForce naming?

Reply:
I agree. But shall be solved in another (new) issue.

Reply:
Issue #2863